### ISSUE TITLE
Code review of tls/mtls feature

### ISSUE THREAD

**Issue by kolbma** [SergioBenitez/Rocket#2082]
I'd like to point you on this suggestion https://github.com/rocket-org/Rocket/issues/4  
Maybe you can support and step in.

**Comment by SergioBenitez** [comment]
I don't think this needs its own issue. Furthermore, the vast majority of the functionality, especially the security sensitive parts, are implemented outside of Rocket. As such, I'm closing this out.
